### DISCUSSION TITLE
Typo error in the Video Compression directory

### DISCUSSION THREAD

**Discussion by Clement-Nicolle** [InterDigitalInc/CompressAI#128] [General] [answered]
Hi !

I'm using your repository to compress some videos and I thing having spotted a typo in the ScaleSpaceFlow class.
Given x a frames list and u,v = model.compress(x), the method model.decompress(u,v) returns the following error :
'ScaleSpaceFlow' object has no attribute 'decoder_inter'

After looking to the ScaleSpaceFlow class, the error seems coming from the line 432 and changing 
x_ref, out_interframe = self.decoder_inter(x_ref, string, shape)
to 
x_ref = self.decode_inter(x_ref, string, shape)
makes the code works fine. (the method decoder doesn't exist contrary to the decode one, and this function returns just one value, not two)

Would it be possible for you to fix this error ?

Thank you in advance for your answer and for making this wonderful repository available. 

Clément

**Comment by fracape** [comment, marked as answer]
Hi, 
Thank you for reporting this. Sorry for the late reply, this is actually more a bug issue than a discussion :).
I think I corrected that in the last version 10ish days ago. Please check and let me know if there is still an issue.
